lesson07/assignment/my-yoga-app/src/components/Home.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+<div>
+
+<div class="grid-container">
+<div class="header">
+<header>
+<div class="logo">
+<img src="./images/yoga-chakra.jpg" id="yoga-chakra"/>
+
+</div>
+<h1>Yoga</h1>
+</header>
+</div>
+<!--navbar-->
+<div class="nav-bar navbar-expand">
+<nav>
+<ul class="nav-links">
+<li v-for="item in items" v-bind:key="item"><a v-bind:href="item.path">{{item.name}}</a></li>
```

Comment:
This is causing the error in the web console when visiting your home page. Keys are unique identifiers we create when dynamically generating code. Keys can only be strings or integers. You'll want to change your key to something like: `v-bind:key="item.name"`or add an ID field to your item data, then use item.id.

Reply:
got it ...thanks